app/assets/locales/locale-en.json

```diff
@@ -1292,6 +1294,13 @@
             "support_block": "For support, please contact XBTS at:",
             "under_construction": "Under Construction"
         },
+        "ioxbank": {
+                "coming_soon": "Coming soon",
```

Comment:
coming soon?

Reply:
That's the order, check other gateways having *coming_soon*.